Lib/fontTools/svgLib/path/shapes.py

```diff
@@ -16,12 +19,25 @@ def _strip_xml_ns(tag):
     return tag.split('}', 1)[1] if '}' in tag else tag
 
 
+def _transform(raw_value):
+    # start simple: if you aren't exactly matrix(...) then no love
+    match = re.match(r'matrix\((.*)\)', raw_value)
+    if not match:
+        raise ValueError('unknown transform %s' % raw_value)
+    matrix = [float(p) for p in re.split(r'\s+|,', match.group(1))]
```

Comment:
you can directly make a tuple here with a generator expression like `tuple(float(p) for p in ...` instead of list comprehension and then `tuple(matrix)` at the end